### ISSUE TITLE
9.5 (33/5) "nonblocking expression" vs "Nonblocking aspect"

### ISSUE THREAD

**Issue by nholsti** [pat-rogers/Ada-202x-WG9-Informal-Review#157]
The phrase "nonblocking expression" should be "Nonblocking aspect", I believe.

**Comment by sttaft** [comment]
Agreed -- it would be more consistent with preceding paragraphs to say "aspect" here instead of "expression."

**Comment by ARG-Editor** [comment]
All of this text originally said "nonblocking expression" (because the aspect represented an expression like a precondition rather than a Boolean value); when this was redone late in the process that was changed and presumably this usage was missed.

There are two other uses that were missed. In 9.5(19/5) there is a long-winded redundant statement starting with "in that case" that I just replaced by "in that case, the aspect for the entity is True".

The third place was the AARM note 9.5(22.a/5), which was rewritten in AI12-0374-2, but that rewrite never made it into the draft AARM. I fixed that now.

These two changes to 9.5 (19/5 & 33/5) were treated as Editorial Reviews in AI12-0374-2, since they don't change any meaning and clearly were intended in the rewrite of AI12-0374-2.
